### ISSUE TITLE
Server Side RPT Error Undefined variable achilles_fnc_setunitammodef

### ISSUE THREAD

**Issue by jokoho48** [ArmaAchilles/AresModAchillesExpansion#107]
i currently dont know in what situation it gets thrown just found it while searching in the server RPT
````
Error in expression <mmo = _this select 1;
{[_x, _ammo] call Achilles_fnc_setUnitAmmoDef} foreach _en>
  Error position: <Achilles_fnc_setUnitAmmoDef} foreach _en>
  Error Undefined variable in expression: achilles_fnc_setunitammodef
```

**Comment by oOKexOo** [comment]
achilles_fnc_setunitammodef should be a public variable and thus also be defined on the server. Maybe there is a case where it doesn't get defined before it's needed. I have to check the corresponding scripts thoroughly.

**Comment by CreepPork** [comment]
@jokoho48 This issue should be resolved in the 0.1.0 branch. I'm not sure completely as you need to test it but it should be resolved. For this reason, I'm closing the issue.
